consensus/e2e_tests/pacemaker_test.go

```diff
@@ -43,89 +43,24 @@ func TestPacemakerTimeoutIncreasesRound(t *testing.T) {
 
 	// Advance time by an amount shorter than the pacemaker timeout
 	advanceTime(t, clockMock, 10*time.Millisecond)
-
-	// Verify consensus started - NewRound messages have an N^2 complexity.
-	numExpectedMsgs := numValidators * numValidators
-	_, err := WaitForNetworkConsensusEvents(t, clockMock, eventsChannel, consensus.NewRound, consensus.Propose, numExpectedMsgs, consensusMessageTimeoutMsec, true)
-	require.NoError(t, err)
-
-	for pocketId, pocketNode := range pocketNodes {
-		assertNodeConsensusView(t, pocketId,
-			typesCons.ConsensusNodeState{
-				Height: 1,
-				Step:   uint8(consensus.NewRound),
-				Round:  0,
-			},
-			GetConsensusNodeState(pocketNode))
-	}
+	height := uint64(1)
+	step := uint8(consensus.NewRound)
+	round := uint8(0)
+	_ = waitForNewRound(t, clockMock, eventsChannel, pocketNodes, height, step, round, numValidators, consensusMessageTimeout, true)
 
 	// Force the pacemaker to time out
 	forcePacemakerTimeout(t, clockMock, paceMakerTimeout)
-
-	// Verify that a new round started at the same height
-	_, err = WaitForNetworkConsensusEvents(t, clockMock, eventsChannel, consensus.NewRound, consensus.Propose, numExpectedMsgs, consensusMessageTimeoutMsec, true)
-	require.NoError(t, err)
-	for pocketId, pocketNode := range pocketNodes {
-		assertNodeConsensusView(t, pocketId,
-			typesCons.ConsensusNodeState{
-				Height: 1,
-				Step:   uint8(consensus.NewRound),
-				Round:  1,
-			},
-			GetConsensusNodeState(pocketNode))
-	}
+	_ = waitForNewRound(t, clockMock, eventsChannel, pocketNodes, height, step, round+1, numValidators, consensusMessageTimeout, true)
```

Comment:
Try to pretend this is your first day at work. What comments would help you understand what's going on? 

Now, always have this mindset.

```suggestion
       // Wait for the round=1 to fail
	_ = waitForNewRound(t, clockMock, eventsChannel, pocketNodes, height, step, round+1, numValidators, consensusMessageTimeout, true)
```